frontend/appflowy_tauri/src/appflowy_app/utils/document/blocks/code/index.ts

```diff
@@ -0,0 +1,39 @@
+import { getPointOfCurrentLineBeginning } from '$app/utils/document/blocks/text/delta';
+import { Editor, Transforms } from 'slate';
+
+export function indent(editor: Editor, distance: number) {
+  const beginPoint = getPointOfCurrentLineBeginning(editor);
+  let emptyStr = '';
+  for (let i = 0; i < distance; i++) {
+    emptyStr += ' ';
+  }
```

Comment:
is there any function can repeat the text with given times?

like, `' ' * distance`, `string.repeat(' ', distance)` or ' '.repeat(distance).

Reply:
You can use pad.

```suggestion
  let emptyStr = ''.padStart(distance)
```